<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QWidget" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LChat</string>
  </property>
  <widget class="QPushButton" name="login_button">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>234</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_LineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>239</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwd_LineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>239</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="auto_login_box">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>230</y>
     <width>80</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>自动登录</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="rem_pwd_box">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>80</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>记住密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>80</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>找回密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TCP_CHAT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>注册账号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="port_line">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>300</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>300</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>设置端口</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
